packages/extension-ui/src/messaging.ts

```diff
@@ -13,6 +12,8 @@ import { metadataExpand } from '@polkadot/extension-chains';
 import allChains from '@polkadot/extension-chains/chains';
 import chrome from '@polkadot/extension-inject/chrome';
 
+import { getSavedMeta, setSavedMeta } from './MetadataCache';
```

Comment:
This is not particularly nice, but it appears that I can't use a `spyOn` test on a function that is not imported. So I had to move the `MetadataCache` out of `messagine`, to import them and be able to spy on them.

Also this `MetadataCache` might be better place somewhere else? I thought about a `./util` but it didn't feel right either.